etc/compile/wait.sh

```diff
@@ -2,8 +2,7 @@
 
 CONTAINER="${1}"
 
-docker wait "$CONTAINER" 2>/dev/null 1>/dev/null
-RET=$?
+RET=`docker wait "$CONTAINER"`
```

Comment:
this was change because `docker wait` prints the exit code, rather than returning the exit code, of the container: https://docs.docker.com/engine/reference/commandline/wait/